cardano-launcher/test/LauncherSpec.hs

```diff
@@ -139,4 +132,69 @@ setWorkingDirectorySpec = describe "Set working directory" $ do
             after <- getCurrentDirectory
             return (done, before, after)
         assert $ not done
-        assert $ before == after
\ No newline at end of file
+        assert $ before == after
+
+configurationSpec :: Spec
+configurationSpec = describe "configuration files" $ modifyMaxSuccess (const 1) $ do
+    mapM_ testLauncherParsable launcherDatas
```

Comment:
You can try using the supported functionality for that - https://hspec.github.io/writing-specs.html#using-hooks

Reply:
I have read the document and have no idea what this is about. Can you try to explain what the problem is and how hooks can solve the issue?

Reply:
If all the tests have something in common that you set up before or after, you can use hooks to remove that duplication. But it's good as it is now.